bin/deployment/test_data_importer.py

```diff
@@ -727,16 +727,18 @@ def create_activation_keys(cp, owners):
     activation_keys = []
 
     # Build a set of activation key data to use to generate our keys
-    for owner_key in owners:
+    for owner_key, owner in owners.items():
+        # If owner(organization) is in Entitlement mode skip it
+        if owner["contentAccessMode"] != "entitlement":
+            continue
+
         activation_keys.append((owner_key, "default_key", None))
         activation_keys.append((owner_key, "awesome_os_pool", None))
 
         pools = cp.list_pools(owner_key)
 
         for pool in pools:
-            key_name = '{owner_key}-{prod_id}-key-{pool_id}'.format(
-                owner_key=owner_key, prod_id=pool['productId'], pool_id=pool['id'])
-
+            key_name = f"{owner_key}-{pool['productId']}-key-{pool['id']}"
```

Comment:
not a bad change, however it is not related to this PR :) so move it out of this commit, please

Reply:
droped